templates/gallery/gallery-map.html

```diff
@@ -0,0 +1,318 @@
+<!doctype html>
+
+<!---
```

Comment:
copyright should be the first thing on the file

Reply:
I'll do this, and while it shouldn't break content detection for most clients (https://mimesniff.spec.whatwg.org/ lists `<!--` as an indication that the document's content type is HTML), it's possible it'll break some.